plugins/web/opentelemetry-plugin-react-load/src/BaseOpenTelemetryComponent.ts

```diff
@@ -58,7 +58,7 @@ export class BaseOpenTelemetryComponent extends React.Component {
   static setTracer(name: string, version?: string): void {
     BaseOpenTelemetryComponent._tracer = api.trace.getTracer(
       name,
-      version ? version : VERSION
+      version ? version : PACKAGE_VERSION
```

Comment:
I think we can also update the line higher up in the class for `readonly component: string = 'react-load';` to instead be `readonly component: string = PACKAGE_NAME;`

Reply:
same comment as above - I support doing this as a dedicated `fix` PR which only targets the name of the instrumentation (which is an end user public API content).